posts/quarto_migration/index.md

```diff
@@ -0,0 +1,51 @@
+
+We were quickly convinced that Quarto was the right choice after some experimentation: the user experience was smooth, their documentation was very clear and all in one place (unlike what we're used to in the Sphinx ecosystem, having to navigate between various extension websites) and it seemed it had all the features we were looking after. The only point that made us hesitate was that Quarto extensions have to be authored in [Lua](https://en.wikipedia.org/wiki/Lua_(programming_language)) and none of us had any experience in that language. Yet we decided that this shouldn't block us and went ahead with the migration.
+
+## Migrating to Quarto
+
+The migration all happened in [this PR](https://github.com/holoviz-dev/blog/pull/27):
+
+- We had to convert the `<post>.ipynb-meta` sidecar files used by the `pelican-jupyter` files to the special header Quarto needs at the beginning of every document.
+- In the notebooks themselves we had very little changes to make, except to handle nested and indented raw HTML included in Markdown cells that wasn't displayed as HTML by Quarto but partially wrapped in a `<code>` HTML element, removing the indentation fixed that (wrapping it in `:::{=html} <... :::` would also have worked).
+- Since we had to move all the posts to the `/posts` directory, the links to our old blog posts changed. We set up redirect links using the `alisases` document option to preserve these old links.
+- For a change compared to our old blog we went for the *default* listing layout instead of *grid*.
+- We customised a little the styling to attempt to align it with the styling of other HoloViz websites.
+
+![Quarto version of the blog](./images/new_site.png)
+
+While the migration went well and was pretty quick to complete, we listed [a few issues](https://github.com/holoviz-dev/blog/issues/34) that we might fix in future iterations. However we're not too surprised by experiencing some problems, our blog posts often contain a lot of complex HMTL and Javascript bits that aren't necessarily easy to handle.
+
+## Easier contribution
+
+Moving to Quarto improved the contributor experience, with a solid [VSCode extension](https://quarto.org/docs/tools/vscode.html) and a nice and fast *preview* mode, and again their excellent documentation.
+
+We also made our infrastructure easier to manage which improved the contributor experience:
+
+- the site is no longer hosted on AWS but on Github Pages
+- a [development version](https://holoviz-dev.github.io/blog-dev/) has been deployed, it is re-built and re-deployed automatically on every *Pull Request* event
+- the [main site](https://blog.holoviz.org/) is re-built and re-deployed whenever a *Pull Request* is merged
+
+If you feel like contributing to the HoloViz blog, head over to its [Github repo](https://github.com/holoviz-dev/blog) and follow the instructions!
```

Comment:
:) Maybe @sophiamyang can demo how to contribute a HoloViz blog in a separate  blog hah